INSTALL.md

```diff
@@ -181,11 +181,11 @@ cd pokecrystal
 To build **pokecrystal.gbc**:
 
 ```bash
-make
+make -j4
```

Comment:
How about `make -j\`nproc\``?

Reply:
(Or `make -j$(nproc)`, either works.)

Reply:
I was just following the precedent set by pokeruby, and this is a sane, easy to remember default. How about adding [this section](https://github.com/pret/pokeruby/blob/master/INSTALL.md#parallel-build) or at least linking to it, instead?

Reply:
Yeah, I guess `-j4` is a good compromise between minimal-but-slow `make` and optimal-but-long `-j$(nproc)`. Please do add a section explaining the multiple and parallel builds like pokeruby has.

Reply:
This depends on the amount of processors your CPU has. Would be more useful to go the nproc route imo